src/hotspot/cpu/ppc/ppc.ad

```diff
@@ -9075,7 +9075,7 @@ instruct lShiftI_andI_immInegpow2_imm5(iRegIdst dst, iRegIsrc src1, immInegpow2
   ins_encode %{
     long src2      = $src2$$constant;
     long src3      = $src3$$constant;
-    long maskbits  = src3 + log2_long((jlong) (julong) (juint) -src2);
+    long maskbits  = src3 + exact_ilog2((jlong) (julong) (juint) -src2);
```

Comment:
I can't verify that this is correct. Someone else has to review this and similar changes below.